src/crimson/thread/ThreadPool.cc

```diff
@@ -0,0 +1,60 @@
+#include "ThreadPool.h"
+#include "crimson/net/Config.h"
+#include "include/intarith.h"
+
+namespace ceph::thread {
+
+ThreadPool::ThreadPool(size_t n_threads,
+                       size_t queue_sz)
+  : queue_size{round_up_to(queue_sz, std::thread::hardware_concurrency())},
```

Comment:
(looks like my last comment raced with your push)

seastar can be configured to run with a different number of reactor threads, so i think this math should use `seastar::smp::count` instead

Reply:
thanks and fixed!